library/tests/src/com/google/maps/android/BiMultiMapTest.java

```diff
@@ -7,37 +7,37 @@
 
 public class BiMultiMapTest extends TestCase {
 
-    private BiMultiMap<String> mMap = new BiMultiMap<>();
-
     public void testSingle() {
+        BiMultiMap<String> mMap = new BiMultiMap<>();
```

Comment:
You should rename this, it's no longer a member so should not have the m prefix.